contracts/src/invoke/Issuance.sol

```diff
@@ -44,8 +44,16 @@ contract Issuance {
 
         require(tokens.length > 0, "No tokens in vault");
 
+        uint256 amountIncludingIntradayInflation = fmul(
```

Comment:
hmm I dont think i understand this chagne:

 real units should already be using current multiplier? which implicitly includes the intraday infaltion, I think this is effectivly charge them twice (for intraday)

`realUnits()` is scaling down the units to match intraday, THEN `amountIncludingIntradayInflation` is also scaling UP the amount to try to account for this